mm/kfence.c

```diff
@@ -545,6 +559,10 @@ int alloc_kmem_cache_cpus(struct kmem_cache *s);
 
 void __init kfence_init(void)
 {
+	if (!kfence_sampling_rate) {
+		pr_info("kfence disabled. To enable, run with kfence_sampling_rate != 0\n");
```

Comment:
If the kernel is compiled with KFENCE, and it's not enabled by default, I'd rather we not say anything to avoid inconveniencing people. Presumably, because if it is disabled, people shouldn't notice the difference.

If it is enabled, we should say so after it was initialized, as now people may want to know about it if their system behaves differently.

Reply:
On the other hand, if a user compiles the kernel with KFENCE, they are expecting it to work, and may not notice that an extra parameter is required, unless we tell them.

Reply:
True enough. Then it should print something in both cases.

Reply:
Ah, but we have the default sampling rate for that case, so should be fine. I'll remove the message.